### ISSUE TITLE
use QAccessibleAnnouncementEvent in addition to available options

### ISSUE THREAD

**Issue by beqabeqa473** [BearWare/TeamTalk5#2283]
in qt6.8, QAccessibleAnnouncementEvent was introduced to send some text to screenreaders.

It covers all three desktop operating systems, linux, macOS, windows.

It would be great to add a possibility to switch to native qt QAccessibleAnnouncementEvent from settings.

I think it is a good way to go for linux and macOS, as i remember, teamtalk cannot output text to screenreader on those platforms.

**Comment by beqabeqa473** [comment]
@bear101 please do not close this issue to easily track progress.
This should be included in teamtalk to improve behaviour on other platforms rather than windows, however i think this should be enabled for windows as well.

**Comment by peter1384** [comment]
please also implement this for windows as wel, for those who don't want a tolk usage, or those who the tolk doesn't work for them, so they can stil get tts's with their screenreaders normally. Thanks

**Comment by CoBC** [comment]
It's exactly what we say, we'll implement this for all platforms, so, sorry, but your comment is really useless here.

**Comment by peter1384** [comment]
@CoBC ok, i'm sorry. I was thinking that you are only gonna implement this for mack and linux because you think this is not needed for windows, so i wanted to comment and say we have problems linking tolk to teamtalk, so please implement this for windows as wel. But if it's already the case, Thank you.
